src/inc/conattrs.hpp

```diff
@@ -17,11 +17,12 @@ enum class CharacterAttributes : uint16_t
     Blinking = 0x04,
     Invisible = 0x08,
     CrossedOut = 0x10,
-    Underlined = 0x20,
-    DoublyUnderlined = 0x40,
-    Faint = 0x80,
-    Unused1 = 0x100,
-    Unused2 = 0x200,
+    Faint = 0x20,
+
+    // 7th, 8th, 9th bit reserved for Underline styles
+    UnderlineStyle = 0x1C0,
+
+    DoublyUnderlined = 0x200,

```

Comment:
we may want to note here why we're keeping it separate, for future maintainers

Reply:
I thought we agreed this was going to be dropped. We don't need to track this separately.

Reply:
Oh, I see. I was thinking about it from the angle of probably wanting ConPTY to preserve whether a double underline came in from an extended underline style. Terminal emulators that support double underline might not support `4:2` (but they should!).

I need to go read up on the rest of that discussion. Sorry :)

Reply:
> Terminal emulators that support double underline might not support `4:2`

Yeah, but that's why we should be passing through the underline state in the most compatible way possible, regardless of how it was originally received. So it doesn't matter if it was original set with `4:2` or with `21` - both should be passed through to conpty as `21`. Same reason we pass through the RGB colors with semicolons, regardless of how they were originally received.

Reply:
ACKed. Sounds right to me. We shouldn't need to track DoublyUNderlined separately then. Having read up on the prior discussion, I doubly agree.

Reply:
There's no trace of `CharacterAttributes::DoublyUnderlined` anymore 😅. Resolving.